src/components/FormSelectList.vue

```diff
@@ -557,9 +573,19 @@ export default {
             ? Mustache.render(this.options.value, item)
             : Mustache.render(`{{${this.options.value || "content"}}}`, item);
 
+        // Modified ariaLabel handling        
+        let itemAriaLabel = itemContent;
```

Comment:
maybe will be better to create a new reusable method  
example 
 ```
const renderWithMustache = (template, data, defaultKey) => {
    return template && template.includes("{{")
      ? Mustache.render(template, data)
      : Mustache.render(`{{${template || defaultKey}}}`, data);
  };
```

in order to handle all mustache templates, so we can avoid repeated code fragments